docker-compose.yml

```diff
@@ -27,6 +44,7 @@ services:
       - temporal
     environment:
       - TEMPORAL_CLI_ADDRESS=temporal:7233
+      - TEMPORAL_PERMIT_WRITE_API=true
```

Comment:
what is this env?

Reply:
this activates the Terminate button in temporal web, makes it full CRUD

Reply:
i think i pasted it in the wrong service